<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="e1441e9e-85e7-404d-a13e-3fca73c383e2" name="Admin@D45 2022-03-10 08:48:47" runUser="D45\Admin" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="904e1636-c7f5-45e3-b84f-b9dab1cc98be">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Admin_D45 2022-03-10 08_48_47">
      <DeploymentItem filename="C:\Users\Admin\Desktop\Calculator\Calculator\Calculator\bin\Debug\Calculator.exe" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2022-03-10T08:48:47.6910000+07:00" queuing="2022-03-10T08:48:47.7110000+07:00" start="2022-03-10T08:48:47.7140000+07:00" finish="2022-03-10T08:48:47.8690000+07:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="TestWithDataSource" storage="c:\users\admin\desktop\calculator\calculator\calculatortester\bin\debug\calculatortester.dll" id="b16e3180-a64c-507c-c325-e19aaafbe832">
      <Execution id="714db056-8a2e-4b15-98a6-3de7fcc94148" />
      <TestMethod codeBase="C:/Users/Admin/Desktop/Calculator/Calculator/CalculatorTester/bin/Debug/CalculatorTester.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="CalculatorTester.UnitTest2, CalculatorTester, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TestWithDataSource" />
      <DataSource connectionString=".\CalculatorTester\Data\TestData.csv" providerInvariantName="Microsoft.VisualStudio.TestTools.DataSource.CSV" tableName="TestData#csv" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="b16e3180-a64c-507c-c325-e19aaafbe832" executionId="714db056-8a2e-4b15-98a6-3de7fcc94148" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="714db056-8a2e-4b15-98a6-3de7fcc94148" testId="b16e3180-a64c-507c-c325-e19aaafbe832" testName="TestWithDataSource" computerName="D45" startTime="2022-03-10T08:48:47.7190000+07:00" endTime="2022-03-10T08:48:47.8580000+07:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="714db056-8a2e-4b15-98a6-3de7fcc94148" resultType="DataDrivenTest">
      <Output>
        <ErrorInfo>
          <Message>The unit test adapter failed to connect to the data source or to read the data. For more information on troubleshooting this error, see "Troubleshooting Data-Driven Unit Tests" (http://go.microsoft.com/fwlink/?LinkId=62412) in the MSDN Library.
Error details: 'C:\Users\Admin\Desktop\Calculator\Calculator\TestResults\Admin_D45 2022-03-10 08_48_47\Out\CalculatorTester\Data' is not a valid path.  Make sure that the path name is spelled correctly and that you are connected to the server on which the file resides.</Message>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>